config/default.json

```diff
@@ -38,5 +38,8 @@
 	},
 	"CTL_TOOLS": {
 		"EXTERNAL_TOOL_MAX_LOGO_SIZE_IN_BYTES": 300000
+	},
+	"ADMIN_API": {
+		"MODIFICATION_THRESHOLD": 60000
```

Comment:
if your end var represents a temporal unit, add the temporal unit!

bad: TIME_TO_GO_HOME=20
good: HOUR_OF_DAY_TO_GO_HOME=20

or

bad: CRON_JOB_INTERVAL=15
good: CRON_JOB_INTERVAL_MIN=15

Reply:
done